### DISCUSSION TITLE
CPC200-UZW Plus not powering on after update

### DISCUSSION THREAD

**Discussion by Kavamaro** [ludwig-v/wireless-carplay-dongle-reverse-engineering#130] [Q&A]
Hello.

I started the update for the carplay dongle and it was completed successfully but the device wont startup.
Please check the attached images for the update process and feedback how to fix it by offline update or rollback.
i downloaded the roll back img but no luck as the device not powering when i plug it to a charger.![image](https://user-images.githubusercontent.com/97364420/155418609-a03d943a-db58-4e5f-9be0-8c2031cd6ec6.png)![image](https://user-images.githubusercontent.com/97364420/155418632-4a76ceaf-6318-4caf-9641-b66949976afe.png)

**Comment by dillan161** [comment]
I have the same problem just now, my  is not powering on , any recommendation ?

**Comment by williamccc** [comment]
Follow the instruction here:
https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/72#discussioncomment-2315150

For the flash dump file, use MX25L12835F_CFW_20210830.BIN.

**Reply by Converseallstar95** [reply]
> Follow the instruction here: [#72 (reply in thread)](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/72#discussioncomment-2315150)
> 
> For the flash dump file, use MX25L12835F_CFW_20210830.BIN.

Wondering if you could shed any light on my issues, I'm new to this adapter. Thanks!
https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/72#discussioncomment-2570216
